pkgs/misc/dxvk/darwin-dxvk-compat.patch

```diff
@@ -1,35 +1,48 @@
      D3D11CommonShader module;
- 
+
 -    if (!m_dxvkDevice->features().extTransformFeedback.transformFeedback)
 -      return DXGI_ERROR_INVALID_CALL;
-+      if (!m_dxvkDevice->features().extTransformFeedback.transformFeedback) {
-+          Logger::err(
-+                   "D3D11: CreateGeometryShaderWithStreamOutput:"
-+                   "\n  Transform feedback not supported by device");
-+          return S_OK;
-+      }
- 
++    if (!m_dxvkDevice->features().extTransformFeedback.transformFeedback) {
++        Logger::err(
++                 "D3D11: CreateGeometryShaderWithStreamOutput:"
++                 "\n  Transform feedback not supported by device");
++        return S_OK;
++    }
+
      // Zero-init some counterss so that we can increment
      // them while walking over the stream output entries
-@@ -1973,8 +1977,8 @@ namespace dxvk {
+@@ -1920,7 +1924,7 @@ namespace dxvk {
+     DxvkDeviceFeatures supported = adapter->features();
+     DxvkDeviceFeatures enabled   = {};
+
+-    enabled.core.features.geometryShader                          = VK_TRUE;
++    enabled.core.features.geometryShader                          = supported.core.features.geometryShader;
+     enabled.core.features.robustBufferAccess                      = VK_TRUE;
+     enabled.core.features.shaderStorageImageWriteWithoutFormat    = VK_TRUE;
+     enabled.core.features.depthBounds                             = supported.core.features.depthBounds;
+@@ -1951,7 +1955,7 @@ namespace dxvk {
+       enabled.core.features.sampleRateShading                     = VK_TRUE;
+       enabled.core.features.samplerAnisotropy                     = supported.core.features.samplerAnisotropy;
+       enabled.core.features.shaderClipDistance                    = VK_TRUE;
+-      enabled.core.features.shaderCullDistance                    = VK_TRUE;
++      enabled.core.features.shaderCullDistance                    = supported.core.features.shaderCullDistance;
+       enabled.core.features.textureCompressionBC                  = VK_TRUE;
+       enabled.extDepthClipEnable.depthClipEnable                  = supported.extDepthClipEnable.depthClipEnable;
+       enabled.extHostQueryReset.hostQueryReset                    = VK_TRUE;
+@@ -1971,8 +1975,8 @@ namespace dxvk {
        enabled.core.features.logicOp                               = supported.core.features.logicOp;
        enabled.core.features.shaderImageGatherExtended             = VK_TRUE;
        enabled.core.features.variableMultisampleRate               = supported.core.features.variableMultisampleRate;
 -      enabled.extTransformFeedback.transformFeedback              = VK_TRUE;
 -      enabled.extTransformFeedback.geometryStreams                = VK_TRUE;
-+        enabled.extTransformFeedback.transformFeedback              = supported.extTransformFeedback.transformFeedback;
-+        enabled.extTransformFeedback.geometryStreams                = supported.extTransformFeedback.geometryStreams;
```

Comment:
The diff view of github is very confusing here...

Reply:
The previous version of the patch had some whitespace issues that didn’t match the surrounding style. I made the changes to DXVK manually then regenerated it with `git diff` for the new version. Should I rework to keep the diff noise down?

Reply:
Na, GitHub is just poor ad displaying diffs diff.